apps/safe-claiming-app/src/components/helpers/UnexpectedError.tsx

```diff
@@ -20,7 +20,8 @@ export const UnexpectedError = ({ error }: { error: string }) => {
           color: ({ palette }) => palette.error.main,
         }}
       >
-        Unexpected Error while loading the app: {error}
+        Unexpected Error while loading the app:{" "}
```

Comment:
Why is "Error" capitalized?

Reply:
I don't think there's a reason for such. Might have been a small mistake? @schmanu